client/src/Components/ShippingInformation.jsx

```diff
@@ -0,0 +1,186 @@
+import React from 'react';
+
+class ShippingInformation extends React.Component{
+  constructor(props){
+    super(props)
+    this.state = {
```

Comment:
You are duplicating state variables, this is bad. When we separate concern, "state" is one of those concerns, and should be maintained in a single file and passed down as props.